BlazorSodium/Sodium/PublicKeySignature.cs

```diff
@@ -0,0 +1,33 @@
+﻿using BlazorSodium.Sodium.Models;
+using System;
+using System.Runtime.InteropServices.JavaScript;
+using System.Text.Json;
+
+namespace BlazorSodium.Sodium
+{
+   public static partial class PublicKeySignature
```

Comment:
This is a really sloppy implementation.  I'm asking around to see if there is a cleaner option.